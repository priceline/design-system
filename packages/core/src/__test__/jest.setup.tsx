import '@testing-library/jest-dom'
import 'jest-styled-components'

global.requestAnimationFrame =
  global.requestAnimationFrame ||
  function _raf(cb) {
    return setTimeout(cb, 0)
  }
