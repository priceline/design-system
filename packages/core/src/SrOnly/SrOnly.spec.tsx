import { render } from '../__test__/testing-library'
import { SrOnly } from './SrOnly'

describe('SrOnly', () => {
  test('renders with approprite styles to display offscreen', () => {
    const { asFragment } = render(<SrOnly />)
    expect(asFragment()).toMatchSnapshot()
  })
})
