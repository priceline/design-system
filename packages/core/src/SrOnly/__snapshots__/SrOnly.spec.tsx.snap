// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SrOnly renders with approprite styles to display offscreen 1`] = `
<DocumentFragment>
  .c0 {
  font-family: 'Montserrat','Helvetica Neue',Helvetica,Arial,sans-serif;
  line-height: 1.4;
  font-weight: 500;
}

.c0 * {
  box-sizing: border-box;
}

.c1 {
  position: absolute;
  overflow: hidden;
  -webkit-clip: rect(0 0 0 0);
  clip: rect(0 0 0 0);
  height: 1px;
  width: 1px;
  margin: -1px;
  padding: 0;
  border: 0;
}

<div
    class="c0"
  >
    <span
      class="c1"
    />
  </div>
</DocumentFragment>
`;
